.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,170 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - dev
+      - rb/parallelize-ci-workflow
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build_and_publish_image:
+    name: Build and Publish Image
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: pgc-images.sbgenomics.com/d3b-bixu/open-pedcan
+          tags: |
+            type=raw,value=analysisjob
+            # Only tag the image with latest if we're building on the default
+            # branch (e.g., dev).
+            type=raw,value=latest,enable={{is_default_branch}}
```

Comment:
The [Docker metadata action](https://github.com/docker/metadata-action) generates the image tags used in the build and push step. We tag each image as `analysisjob` and only tag an image as `latest` if the `dev` branch triggered the workflow.

See: https://github.com/docker/metadata-action#is_default_branch